src/components/tooltip/Tooltip.tsx

```diff
@@ -212,6 +215,7 @@ Tooltip.defaultProps = {
 };
 
 Tooltip.propTypes = {
+  children: PropTypes.any,
```

Comment:
Can we get the correct type instead of `any`?

Reply:
Updated to `PropTypes.node` 👍